Fix the following error due to jammy's older version of libuv:

../src/node_platform.cc: In function 'int node::{anonymous}::GetActualThreadPoolSize(int)':
../src/node_platform.cc:50:24: error: 'uv_available_parallelism' was not declared in this scope
   50 |     thread_pool_size = uv_available_parallelism() - 1;
      |                        ^~~~~~~~~~~~~~~~~~~~~~~~

--- a/src/node_platform.cc
+++ b/src/node_platform.cc
@@ -47,7 +47,7 @@ static void PlatformWorkerThread(void* d
 
 static int GetActualThreadPoolSize(int thread_pool_size) {
   if (thread_pool_size < 1) {
-    thread_pool_size = uv_available_parallelism() - 1;
+    thread_pool_size = (int)sysconf(_SC_NPROCESSORS_ONLN) - 1;
   }
   return std::max(thread_pool_size, 1);
 }
